Sources/QueuesRedisDriver/JobsRedisDriver.swift

```diff
@@ -62,7 +62,11 @@ struct _QueuesRedisQueue {
     let context: QueueContext
 }
 
-extension _QueuesRedisQueue: RedisClient {
+extension _QueuesRedisQueue: RedisClient {    
+    var isConnected: Bool {
+        return true
```

Comment:
I think for now, you can just forward the `self.client.isConnected`, even though it too is just returning `true` statically.